src/puppetlabs/puppetdb/cheshire.clj

```diff
@@ -19,6 +19,12 @@
   "Non-memoize version of add-common-json-encoders!"
   []
   (generate/add-encoder
+   org.postgresql.util.PGobject
+   (fn [data jsonGenerator]
+     (if (.getPrettyPrinter jsonGenerator)
```

Comment:
This needs a comment about why getPrettyPrinter may return null or not, and what that means. and a type annotation on jsonGenerator would be nice too.